x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -63,6 +67,16 @@ export class UptimePlugin
       });
     }
 
+    // This doesn't actually work
```

Comment:
```suggestion
```

I think this works now.. if I should be able to complete the testing instructions if it's working then it's good AFAICT.